src/admin/components/forms/field-types/Array/index.tsx

```diff
@@ -24,6 +24,7 @@ import { scrollToID } from '../../../../utilities/scrollToID';
 import HiddenInput from '../HiddenInput';
 
 import './index.scss';
+import RowLabel from '../../RowLabel';
```

Comment:
Just a code style thing - try and make sure that you group your new imports, and that the `import './index.scss'` is the last import.